openapi2kong/validator.go

```diff
@@ -65,9 +65,19 @@ func generateParameterSchema(operation *v3.Operation, path *v3.PathItem,
 
 	result := make([]map[string]interface{}, len(combinedParameters))
 	i := 0
+	invalidParamCounts := 0
 
 	for _, parameter := range combinedParameters {
 		if parameter != nil {
+			if parameter.In == "cookie" {
+				logbasics.Info(`cookie parameters are not supported for request-validator plugin; 
+					choose either path, query or header`)
```

Comment:
these backticks will preserve the whitespace and newline for the log message, which is not what we want. Also the message can be a bit clearer about the consequences I think, how about `"cookie parameters are not supported by the request-validator plugin; validation will be skipped"`